BRPeerManager.h

```diff
@@ -37,7 +37,7 @@
 extern "C" {
 #endif
 
-#define PEER_MAX_CONNECTIONS 3
+#define PEER_MAX_CONNECTIONS 8
```

Comment:
@andhikayuana : I love that your found this as and way to dial in. Did you test or is this the first change?

Reply:
this is first change